egs/swbd/s5c/local/run_asr_segmentation.sh

```diff
@@ -0,0 +1,145 @@
+                          # is  merged with the previous.
+
+# List of weights on labels obtained from alignment, 
+# labels obtained from decoding and default labels in out-of-segment regions
+merge_weights=1.0,0.1,0.5
+
+prepare_targets_stage=-10
+nstage=-10
+train_stage=-10
+test_stage=-10
+
+affix=_1a
+stage=-1
+nj=80
+
+. path.sh
+. cmd.sh 
+
+set -e -u -o pipefail
+. utils/parse_options.sh 
+
+if [ $# -ne 0 ]; then
+  exit 1
+fi
+
+dir=exp/segmentation${affix}
+mkdir -p $dir
+
+# See $lang/phones.txt and decide which should be garbage
+garbage_phones="lau spn"
+silence_phones="nsn SIL"
+
+for p in $garbage_phones; do 
+  for affix in "" "_B" "_E" "_I" "_S"; do
+    echo "$p$affix"
+  done
+done > $dir/garbage_phones.txt
+
+for p in $silence_phones; do 
+  for affix in "" "_B" "_E" "_I" "_S"; do
+    echo "$p$affix"
+  done
+done > $dir/silence_phones.txt
+
+if ! cat $dir/garbage_phones.txt $dir/silence_phones.txt | \
+  steps/segmentation/internal/verify_phones_list.py $lang/phones.txt; then
+  echo "$0: Invalid $dir/{silence,garbage}_phones.txt"
+  exit 1
+fi
+
+whole_data_dir=${data_dir}_whole
+
+if [ $stage -le 0 ]; then
+  utils/data/convert_data_dir_to_whole.sh $data_dir $whole_data_dir
+fi
+
+###############################################################################
+# Extract features for the whole data directory
+###############################################################################
+if [ $stage -le 1 ]; then
+  steps/make_mfcc.sh --cmd "$train_cmd" --nj $nj --write-utt2num-frames true \
+    ${whole_data_dir} || exit 1
+fi
+
+###############################################################################
+# Get feats for the manual segments
+###############################################################################
+if [ $stage -le 2 ]; then
+  if [ ! -f ${data_dir}/segments ]; then
+    utils/data/get_segments_for_data.sh $data_dir > $data_dir/segments
+  fi
+  utils/data/subsegment_data_dir.sh $whole_data_dir ${data_dir}/segments ${data_dir}/tmp
+  cp $data_dir/tmp/feats.scp $data_dir
+  awk '{print $1" "$2}' $data_dir/segments > $data_dir/utt2spk
+  utils/utt2spk_to_spk2utt.pl $data_dir/utt2spk > $data_dir/spk2utt
+fi
+
+if [ $stage -le 3 ]; then
+  steps/segmentation/prepare_targets_gmm.sh --stage $prepare_targets_stage \
+    --train-cmd "$train_cmd" --decode-cmd "$decode_cmd" \
+    --nj 80 --reco-nj 40 --lang-test $lang_test \
+    --garbage-phones-list $dir/garbage_phones.txt \
+    --silence-phones-list $dir/silence_phones.txt \
+    $lang $data_dir $whole_data_dir $sat_model_dir $model_dir $dir
+fi
+
+if [ $stage -le 4 ]; then
+  utils/copy_data_dir.sh ${whole_data_dir} ${whole_data_dir}_hires_bp
+  steps/make_mfcc.sh --mfcc-config conf/mfcc_hires_bp.conf --nj 40 \
+    ${whole_data_dir}_hires_bp
+  steps/compute_cmvn_stats.sh ${whole_data_dir}_hires_bp
+fi
+
+if [ $stage -le 5 ]; then
+  # Train a TDNN-LSTM network for SAD
+  local/segmentation/tuning/train_lstm_asr_sad_1a.sh \
```

Comment:
The scripts are same as the one in babel. I copied it to switchboard in the new PR #1838.